.ci/common.sh

```diff
@@ -48,7 +48,15 @@ check_core_dumps() {
 }
 
 setup_deps() {
+	sudo pip --upgrade pip
 	sudo pip install neovim
+
+	# For pip3
+	# https://github.com/travis-ci/travis-ci/issues/1528 
+	curl http://python-distribute.org/distribute_setup.py | sudo python3
```

Comment:
Good idea, perhaps this could be done too:
```
sudo python3 <(curl -Ss http://python-distribute.org/distribute_setup.py)
```
I'm not sure if it needs to be quoted... damn I hate shell scripting :)